src/Sulu/Component/Content/Document/Subscriber/CopyLocaleSubscriber.php

```diff
@@ -163,7 +163,7 @@ public function checkPageTreeRoute(RoutableBehavior $destDocument, StructureBeha
         $pageTreeRoutePropertyName = $this->getPageTreeRoutePropertyName($document);
         $routePath = $documentStructure[$pageTreeRoutePropertyName] ?? null;
 
-        if (!$routePath || !\array_key_exists('page', $routePath)) {
+        if (!$routePath || !\is_array($routePath) || !\array_key_exists('page', $routePath)) {
```

Comment:
@Prokyonn added this to avoid: https://github.com/sulu/sulu/pull/7450#issuecomment-2154662081

Reply:
added a test for it, although I'm not exactly sure how it can happen in production 🤔

Reply:
I just created an article in the sulu demo and did a language copy.